<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HisDataTemp</class>
 <widget class="QWidget" name="HisDataTemp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTableView" name="tv_Temp365">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>110</y>
     <width>320</width>
     <height>290</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushCleanButton">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>420</y>
     <width>98</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>清空</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushExitButton">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>420</y>
     <width>98</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="QTableView" name="tv_Temp445">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>110</y>
     <width>320</width>
     <height>290</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
